service/service.go

```diff
@@ -11,14 +11,14 @@ import (
 )
 
 const (
-	lfpFileNamePrefix string = "CHS-LFP-CARD-ERRORS-"
+	ppsFileNamePrefix string = "CHS-PPS-CARD-ERRORS-"
```

Comment:
This prefix might need a steer from Hamayan / Finance as to what they want here.
I find it strange it doesn't say E5 in the file name.
I would suggest this
```suggestion
	ppsFileNamePrefix string = "CHS-PENALTY-PAYMENT-E5-ERRORS-"
```